package.json

```diff
@@ -48,6 +48,7 @@
   "name": "swaps-service",
   "scripts": {
     "bcoin_bcoinregtest": "tap test/bcoin_bcoinregtest/*.js",
+    "bitcoind_regtest": "tap test/core_regtest/*.js",
```

Comment:
bitcoind is a good naming vs core - so the directory should probably be bitcoind too like bitcoind_regtest